db/migrate/20221011125254_create_vacations.rb

```diff
@@ -0,0 +1,14 @@
+class CreateVacations < ActiveRecord::Migration[7.0]
+  def change
+    create_table :vacations do |t|
+      t.date :start_date, null: false
+      t.date :end_date, null: false
+      t.integer :status, default: 1
```

Comment:
Per #266, shouldn't status start as "pending"?

Also, you could put this in a separate variable, e.g. `PENDING_STATUS = 0` and use `t.integer :status, default: PENDING_STATUS`